angular/projects/admin/src/app/admin/document/document-form/document-form.component.ts

```diff
@@ -24,6 +24,7 @@ export class DocumentFormComponent implements OnInit, OnDestroy, OnChanges {
   @Input() onCancelRoute: string;
 
   private _rootSlug: string;
+  authorIsValid: boolean;
```

Comment:
There is no guarantee that there is a data-field called `author` this will break if you try with `page` or `event` in current test project as they don't have any "author field"